tests/unit/services/suite.service.spec.js

```diff
@@ -42,10 +52,13 @@ describe('SuiteService', () => {
     it('should add an alert on error', () => {
       expect(store.state.alerts.length).to.equal(0);
       const e = new Error('mock error');
-      e.response = {
-        statusText: 'Test Status'
+      const stubClient = {
+        uri: null,
+        query: function() {
+          return Promise.reject(e);
+        }
       };
-      sandbox.stub(axios, 'get').rejects(e);
+      sandbox.stub(SuiteService, 'createGraphqlClient').returns(stubClient);
```

Comment:
The rest here are just changes for tests to match what was changed before :+1: hope that helps you reviewing this.